pom.xml

```diff
@@ -19,11 +19,15 @@
     Additionally, the plug-in provides health reporting and build stability based on these combined results.
   </description>
 
+  <properties>
+    <workflow.version>1.7</workflow.version>
+  </properties>
+
   <dependencies>
     <dependency>
       <groupId>org.jvnet.hudson.plugins</groupId>
       <artifactId>analysis-core</artifactId>
-      <version>1.73-SNAPSHOT</version>
+      <version>1.73-beta-SNAPSHOT</version><!-- TODO: change to 1.73 before release -->
```

Comment:
This beta release is something "fictitious", since it's a step SNAPSHOT only used in downstreams of https://github.com/jenkinsci/analysis-core-plugin/pull/31

It should be replaced at release time (once analysis-core 1.73 is released).